devicetypes/fibargroup/fibaro-dimmer-2-zw5.src/fibaro-dimmer-2-zw5.groovy

```diff
@@ -0,0 +1,498 @@
+                case [1,2]: sendEvent(name: "errorMode", value: "overheat"); break;
+            }; break;
+        case 8:
+            switch (cmd.event) {
+                case 0: sendEvent(name: "errorMode", value: "clear"); break;
+                case 4: sendEvent(name: "errorMode", value: "surge"); break;
+                case 5: sendEvent(name: "errorMode", value: "voltageDrop"); break;
+                case 6: sendEvent(name: "errorMode", value: "overcurrent"); break;
+                case 8: sendEvent(name: "errorMode", value: "overload"); break;
+                case 9: sendEvent(name: "errorMode", value: "loadError"); break;
+            }; break;
+        case 9:
+            switch (cmd.event) {
+                case 0: sendEvent(name: "errorMode", value: "clear"); break;
+                case [1,3]: sendEvent(name: "errorMode", value: "hardware"); break;
+            }; break;
+        default: logging("${device.displayName} - Unknown zwaveAlarmType: ${cmd.zwaveAlarmType}","warn");
+    }
+}
+
+def parse(String description) {
+    def result = []
+    logging("${device.displayName} - Parsing: ${description}")
+    if (description.startsWith("Err 106")) {
+        result = createEvent(
+                descriptionText: "Failed to complete the network security key exchange. If you are unable to receive data from it, you must remove it from your network and add it again.",
+                eventType: "ALERT",
+                name: "secureInclusion",
+                value: "failed",
+                displayed: true,
+        )
+    } else if (description == "updated") {
+        return null
+    } else {
+        def cmd = zwave.parse(description, cmdVersions())
+        if (cmd) {
+            logging("${device.displayName} - Parsed: ${cmd}")
+            zwaveEvent(cmd)
+        }
+    }
+}
+
+def zwaveEvent(physicalgraph.zwave.commands.securityv1.SecurityMessageEncapsulation cmd) {
+    def encapsulatedCommand = cmd.encapsulatedCommand(cmdVersions())
+    if (encapsulatedCommand) {
+        logging("${device.displayName} - Parsed SecurityMessageEncapsulation into: ${encapsulatedCommand}")
+        zwaveEvent(encapsulatedCommand)
+    } else {
+        log.warn "Unable to extract Secure command from $cmd"
+    }
+}
+
+def zwaveEvent(physicalgraph.zwave.commands.crc16encapv1.Crc16Encap cmd) {
+    def version = cmdVersions()[cmd.commandClass as Integer]
+    def ccObj = version ? zwave.commandClass(cmd.commandClass, version) : zwave.commandClass(cmd.commandClass)
+    def encapsulatedCommand = ccObj?.command(cmd.command)?.parse(cmd.data)
+    if (encapsulatedCommand) {
+        logging("${device.displayName} - Parsed Crc16Encap into: ${encapsulatedCommand}")
+        zwaveEvent(encapsulatedCommand)
+    } else {
+        log.warn "Unable to extract CRC16 command from $cmd"
+    }
+}
+
+def zwaveEvent(physicalgraph.zwave.commands.multichannelv3.MultiChannelCmdEncap cmd) {

```

Comment:
Doesn't seem like this would be of any use, since the device never creates any child devices.